### ISSUE TITLE
Importer: Configuration frame - disable non-editable fields

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.Core#1246]
Didn't we agree on to make "edit extra fields" that cannot be edited (e.g., "Species, Organ, etc) disabled?

![image](https://user-images.githubusercontent.com/25903040/128727240-5de3d27f-cabe-47b8-abeb-7fcd7d30ec72.png)

**Comment by georgeDaskalakis** [comment]
I could not reproduce this behaviour in the latest state of the develop branch. Are you trying this in PK-Sim or MoBi? In PK-Sim MW is not available (it is not part of the configuration).

**Comment by georgeDaskalakis** [comment]
I totally agree this would be an improvement for the importer, but it is not a bug and this change could introduce errors (disabling of fields is not plainly visual, if done wrong it could fe disable a field when it should actually be editable). So since we are already on the beta, I suggest to mark it for v11.

**Comment by Yuri05** [comment]
implemented

**Comment by PavelBal** [comment]
No it's not done yet ;)  Low prio however.
